images/runtime/ray/Dockerfile

```
@@ -1,5 +1,127 @@
-FROM rayproject/ray:2.23.0-py39-cu121
+FROM registry.access.redhat.com/ubi9/python-311:latest
 
-# Flash Attention 2 requires PyTorch at installation time
-RUN pip install --no-cache-dir -U pip \
-    torch==2.3.1
+LABEL name="ray-ubi9-py311-cu121" \
+      summary="CUDA 12.1 Python 3.11 image based on UBI9 for Ray" \
+      description="CUDA 12.1 Python 3.11 image based on UBI9 for Ray" \
+      io.k8s.display-name="CUDA 12.1 Python 3.11 base image for Ray" \
+      io.k8s.description="CUDA 12.1 Python 3.11 image based on UBI9 for Ray" \
+      authoritative-source-url="https://github.com/opendatahub-io/distrubuted-workloads" \
```

I think this needs to be removed.
```suggestion
      authoritative-source-url="https://github.com/opendatahub-io/distrubuted-workloads"
```

---

Good catch, updated.